### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by ventistudio**
This is the French version of nova defender, it also includes the translation of the main page of the github (added in addition) this was done with heart for the French-speaking community and before everything for the creator of nova defender!

I remain available if there is a need for support in French on discord (ventistudio)!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,148 +13,151 @@
...
-- Steam: https://steamcommunity.com/id/samuelweil/
-
-## Credits
+Vous êtes également invité à me contacter directement :
+- Discord (préféré) : _samuel
+- Steam : https://steamcommunity.com/id/samuelweil/
+- Rappel : le support "français" n'est pas disponible sauf sous la demannde du créateur de nova defender.
+
+## Crédits
+- [VentiStudio | Hikari Umaishi - Traduction Française](https://discord.gg/kQF7ehCJ3S)
```
Comment (opastudien) [inline README.md:159]: I took the liberty of adding a credit for myself on the github I published

File: README.md [hunk 2]
```diff
@@ -13,148 +13,151 @@
...
+- [Nova Defender ( Officiel )](https://github.com/Freilichtbuehne/nova-defender)
```
Comment (opastudien) [inline README.md:160]: I took the liberty of adding a credit for you on the github that I published

**Review by opastudien — COMMENTED** [review]
No donation links have been modified

**Comment by Freilichtbuehne** [comment]
@ventistudio thank you for the pull request!

Is there a reason you translated the `README.md`?
I would like to preserve the default language for it. Could you create a seperate French version of the readme? That would be awesome!

Like here for example: https://github.com/jonatasemidio/multilanguage-readme-pattern

**Pull request closed without merging**